guides/common/modules/con_managing-dhcp-using-smart-proxies.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Managing_DHCP_Using_Smart_Proxies_{context}"]
+= Managing DHCP Using {SmartProxies}
+
+{Project} can integrate with a DHCP service using your {SmartProxy}.
```

Comment:
Perhaps a writer could review this, but the title is plural (SmartProxies) and this sentence is singular. Should it be consistent? `your {singular}` implies there is one while you can use any number of them.

Reply:
Changed to "SmartProxies" attribute.

Reply:
Please close this thread if you agree @Lennonka